lib/samson/hooks.rb

```diff
@@ -18,6 +19,7 @@ class UserError < StandardError
       :project_permitted_params,
       :before_deploy,
       :after_deploy_setup,
+      :after_deploy_create, # for external plugin
```

Comment:
I cannot use `after_deploy_setup` because it seems has its own purpose and passing different data. It is fired after the job actually execute, after setting up necessary stuff.
```
Samson::Hooks.fire(:after_deploy_setup, dir, stage) 
```
So If I have to store extra fields somewhere around the time `deploy` is saved . maybe `after_deploy_save` a better name?